infrahub_sdk/ctl/exporter.py

```diff
@@ -26,9 +26,9 @@ def dump(
     concurrent: int = typer.Option(
         4,
```

Comment:
Do we want to set the default to `None` and it'll take the SDK defaults? (There might be other changes required for this, but mainly the idea of these options should be overrides when defined, but match SDK defaults if not provided.